frontend/src/ui/theme.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```

Comment:
We will have a massive conflict on this file with [my color PR ](https://github.com/MTES-MCT/monitorfish/pull/1437). Can we put these colors in another props `colorsTemp: {}` while the cleaning is not finished?